scripts/generate_module_packages.py

```
@@ -0,0 +1,270 @@
+                            break
+
+                    if os_major:
+                        key = (module_name, os_major, str(stream))
+                        module_packages[key] = package_names
+                        module_count += 1
+
+    except FileNotFoundError:
+        print(f"ERROR: File not found: {modules_yaml_path}", file=sys.stderr)
+        print("Run ./download_modulemd.sh first to download module metadata", file=sys.stderr)
+        return None
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)